.github/workflows/test-suite.yml

```diff
@@ -54,12 +51,22 @@ jobs:
       - if: false
         run: echo jobs need steps
 
+  day_of_week:
+    runs-on: ubuntu-latest
+    outputs:
+      day: ${{ steps.set_day.outputs.day }}
+    steps:
+      - name: set day
+        id: set_day
+        run: |
+          echo "day=$(date +'%a')"
```

Comment:
knit: normally use convention of uppercase for env.vars like DAY

Reply:
it's ok to do the echo, although not really needed because you can easily see in the execution that test job is getting the env variable with the expected value

Reply:
updated in 9a16a1a16fbb846225446ca94c1a33e314c75fd2